src/arch/ArchHooks/ArchHooks_Unix.cpp

```diff
@@ -408,8 +408,13 @@ void ArchHooks::MountUserFilesystems( const RString &sDirOfExecutable )
 	/* Path to write general mutable user data when not Portable
 	 * Lowercase the PRODUCT_ID; dotfiles and directories are almost always lowercase.
 	 */
+#ifdef IS_FLATPAK
```

Comment:
My quick googling says flatpak *does* have access to $HOME. That would be greatly preferred for consistency over defining a new location.

`--filesystem=~/.itgmania:create` should do the trick.

Reply:
I really did not want to do this, since we'll just continue to follow an outdated standard that shouldn't be around anymore, but if consistency is that important throughout all releases, then yeah I can change the flatpak manifest to explicitly get read/write permission for `$HOME/.itgmania`.

Reply:
$HOME/.itgmania is fine

Reply:
shouldn't we always try `XDG_DATA_HOME`, then fall back to `HOME` if `XDG_DATA_HOME` doesn't exist? shouldn't need extra ifdefs

Reply:
I believe I discussed this with @sukibaby at some point, but if we were to actually move away from putting the dot folder into `$HOME`, it would most likely be something along the lines of.
- check for `$HOME/.itgmania`
- if it exists, use it
- if not, use `$XDG_DATA_HOME/itgmania` (and create the folder if necessary)

Reply:
> Yeah my initial suggestion was to check XDG_DATA_HOME first, see if it exists and is not empty. If it isn't then check ~/.itgmania, and if neither exists to make a directory in ~/.itgmania.
> 
> This way it would not break anything existing but people _could choose to move their directory into XDG_DATA_HOME if they prefer it being there_.


This sounds good to me!